src/views/poi_panel.dot

```diff
@@ -80,6 +80,48 @@
 
       {{= this.PoiBlocContainer.render(this.poi) }}
 
+      {{
+        let pictures;
+        const id = this.poi.id.replace('pj:', '');
+        const imagesBlock = this.poi.blocks.find((block) => block.type === 'images');
+        if (imagesBlock) {
+          pictures = imagesBlock.images.filter(img => img.url !== this.poi.topImageUrl);
+        }
+      }}
+
+      {{? pictures && pictures.length }}
+        <div class="poi_panel__pictures">
+            <h5 class="poi_panel__pictures_title">
+              <i class="icon-icon_camera"></i>
+              {{= _('Photos')}}
+            </h5>
+            <a href="https://www.pagesjaunes.fr/pros/{{= id}}" target="_blank" rel="noopener noreferrer" style="text-decoration: none">
+              <div class="poi_panel__pictures_block" style="background-image: url('{{= pictures[0].url}}')">
```

Comment:
This breaks if the URL contains a single quote `'`.  
`&quot;` should be used instead for all images, see `header.dot` for an example.
  
Maybe we could define a function somewhere to build this style property and avoid this bug in the future ?

Reply:
I thought it would be escaped..